solr/solr-ref-guide/src/script-update-processor.adoc

```diff
@@ -35,19 +35,19 @@ potential security vulnerabilities.
 
 The scripting update processor lives in the contrib module `/contrib/scripting`, and you need to explicitly add it to your Solr setup.
 
-Java 11 and previous versions come with a JavaScript engine called Nashorn, but Java 12 will require you to add your own JavaScript engine.   Other supported scripting engines like
-JRuby, Jython, Groovy, all require you to add JAR files.
-
-
-You can either add the `dist/solr-scripting-*.jar` file into Solr’s resource loader in a core `lib/` directory, or via `<lib>` directives in `solrconfig.xml`:
+You can either add the `dist/solr-scripting-*.jar` file into Solr’s core `lib/` directory, or via `<lib>` directives in `solrconfig.xml`:
```

Comment:
It's more probable someone would use SOLR_HOME/lib than a lib directory on a core.  And FYI `<lib>` directives may be going away or be discouraged.  Let's link to `<<libs.adoc#lib-directories,Lib Directories>>` instead of enumerating how to do this, so we can just maintain this sort of info in one place in the ref guide.